README.md

```diff
@@ -58,20 +58,67 @@ pip install -U dataprep
+
+
+Connector provides a simple programming interface to collect structured data from different Web APIs (e.g., Twitter API, Yelp Fusion API, Spotify API, DBLP API), making web data collection easy and efficient, without requiring advanced programming skills.
+
+
+Do you want to leverage the growing number of websites that are opening their data through public APIs?
+
+
+Connector is for you!  
+
+Let's check out the several benefits that Connector offers:
+
+* <ins>**A unified API:**</ins> You can fetch data using one or two lines of code to get data from many websites.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://sfu-db.github.io/dataprep/user_guide/connector/connector.html"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector_main.gif"/></a>
+
+* <ins>**Auto Pagination:**</ins> Connector automatically does the pagination for you! Just specify the desired number of returned results (argument `_count`) without getting into unnecessary detail about a specific pagination scheme.
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://sfu-db.github.io/dataprep/user_guide/connector/connector.html"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector_pagination.gif"/></a>
+
+* <ins>**Smart API request strategy:**</ins> Through the `_concurrency` argument, Connector simplifies concurrency, issuing API requests in parallel while respecting the API's rate limit policy. 
+
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://sfu-db.github.io/dataprep/user_guide/connector/connector.html"><img src="https://github.com/sfu-db/dataprep/raw/develop/assets/connector_concurrency.gif"/></a>
+
+In [configuration files](https://github.com/sfu-db/DataConnectorConfigs), Connector specifies how to connect with each Web API for data gathering. If you want to connect with any of the APIs mentioned in the table below, with one line of code, you can get the most up-to-date version of the config file from our codebase and use it right away!
+
+Many websites in different domains are currently supported. These are some examples: 
+ 
+Category | Web API |  Auth Method  | Connector Config File(s) | Jupyter Notebook / Tutorial | Description | 
+|---|---|---|---|---|---|
+| Social Media | [Twitter](https://developer.twitter.com/en)  | `OAuth2` |  [Twitter config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/twitter) |  [Twitter Jupyter Notebook](https://github.com/sfu-db/dataprep/blob/develop/examples/DataConnector_Twitter.ipynb) |  API endpoint for Tweets information retrieval. |
+| Music | [Spotify](https://developer.spotify.com/)  | `OAuth2` |  [Spotify config file(s)](https://github.com/sfu-db/DataConnectorConfigs/tree/develop/spotify) |  [Spotify tutorial](https://sfu-db.github.io/dataprep/user_guide/connector/DC_Spotify_tut.html) |  Comprehensive API for retrieving albums, artists, and tracks metadata. |
```

Comment:
Shall we change to the notebook example here to be consistent with other items?

Reply:
@peiwangdb Are you referring to the Spotify notebook? If so, I wasn't able to find a notebook for Spotify in our repo: https://github.com/sfu-db/dataprep/tree/develop/examples. For that reason, a link to the current Spotify tutorial was included.

Reply:
Thanks for the explanation @juandavidospina ! Then I am ok with the current status and an update if the link got updated.

Reply:
@peiwangdb Thank you!